src/features/list-view/styles/ListView.scss

```diff
@@ -20,16 +22,24 @@
         text-transform: none;
         width: 100%;
 
-        .bdl-icon-sort-chevron.bdl-ListView-isSortAsc {
+        &--sortable {
```

Comment:
Please avoid this convention. It makes class names very difficult to search globally.

Reply:
There's a tradeoff between searchability and DRY. The names are an eyesore the other way.